src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/caseupdate/CaseUpdateClientTest.java

```diff
@@ -0,0 +1,263 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate;
+
+import com.github.tomakehurst.wiremock.core.Options;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.CaseClientServiceException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.InvalidCaseDataException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.DocumentType;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.DocumentUrl;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ExceptionRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.OcrDataField;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ScannedDocument;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.CaseUpdateDetails;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.SuccessfulUpdateResponse;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.IntegrationTest;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Wrong lexicographical order for 'uk.gov.hmcts.reform.bulkscan.orchestrator.client.caseupdate.model.response.CaseUpdateDetails' import. Should be before 'uk.gov.hmcts.reform.bulkscan.orchestrator.client.model.request.ScannedDocument'.](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4535703)